pkg/virt-controller/watch/migration.go

```diff
@@ -704,8 +707,29 @@ func (c *MigrationController) handleTargetPodCreation(key string, migration *vir
 	// migration was accepted into the system, now see if we
 	// should create the target pod
 	if vmi.IsRunning() {
-		if err := c.createPDB(key, vmi, migration); err != nil {
-			return err
+		if migrations.MigrationNeedsProtection(vmi) {
+			pdbs, err := pdbs.PDBsForVMI(vmi, c.pdbInformer)
+			if err != nil {
+				return err
+			}
+			if len(pdbs) < 1 {
+				log.Log.Object(vmi).Errorf("Found no PDBs protecting the vmi")
+				return err
+			}
+			pdb := pdbs[0]
+
+			if expandedPdb, err := c.expandPDB(pdb, vmi, migration); err != nil {
+				return err
+			} else {
+				pdb = expandedPdb
+			}
```

Comment:
I don't think there's any value in viewign the result of expanding the pdb. that result won't show the pbd is protected as a result of posting the expansion.

Reply:
I did that for the correctness of `isMigrationProtected()` when it is called just a few lines later. I can make it explicit in `isMigrationProtected` that we expect minAvailable to be 2 and get rid of this.